mdbook-tera-backend/README.md

```diff
@@ -0,0 +1,84 @@
+# [Tera](https://github.com/Keats/tera) backend extension for `mdbook`'s HTML backend
+
+[![Visit crates.io](https://img.shields.io/crates/v/mdbook-i18n-helpers?style=flat-square)](https://crates.io/crates/mdbook-tera-backend)
+[![Build workflow](https://img.shields.io/github/actions/workflow/status/google/mdbook-i18n-helpers/test.yml?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/actions/workflows/test.yml?query=branch%3Amain)
+[![GitHub contributors](https://img.shields.io/github/contributors/google/mdbook-i18n-helpers?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/graphs/contributors)
+[![GitHub stars](https://img.shields.io/github/stars/google/mdbook-i18n-helpers?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/stargazers)
+
+This `mdbook` backend makes it possible to use [tera](https://github.com/Keats/tera)
+templates and expand the capabilities of your books. It works on top of the default HTML
+backend.
```

Comment:
Capitalization is always a difficult topic... here I looked at https://keats.github.io/tera/docs/ and see that the project refer to themselves as "Tera":

```suggestion
This `mdbook` backend makes it possible to use [Tera](https://github.com/Keats/tera)
templates and expand the capabilities of your books. It works on top of the default HTML
backend.
```